src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  transition: 300ms ease;
```

Comment:
Consider adding a 'transform' property to the initial state of the card to ensure smooth transitions when scaling on hover. This can help avoid layout shifts.